File: Ej 21 (Parando Tiempo)/Java/ParandoTiempo.java
// DeathbatO

public class ParandoTiempo{

    public static void main(String[] args) {
        // Ejemplo de uso
        // Suma 5 + 7 y retorna el resultado después de 3 segundos
        sumAsync(5, 7, 3); 
    }

    public static void sumAsync(int a, int b, int seconds) {
        Thread thread = new Thread(() -> {
            try {
                Thread.sleep(seconds * 1000); // Convertir segundos a milisegundos
                int result = a + b;
                System.out.println("La suma es: " + result);
            } catch (InterruptedException e) {
                e.printStackTrace();
            }
        });
        thread.start();
    }
}

File: Ej 21 (Parando Tiempo)/Python/ParandoTiempo.py
## DeathbatO

import threading
import time

def sum_async(a, b, seconds):
    def calculate_sum():
        time.sleep(seconds)
        result = a + b
        print("La suma es:", result)

    thread = threading.Thread(target=calculate_sum)
    thread.start()

# Ejemplo de uso
# Suma 5 + 7 y retorna el resultado después de 3 segundos
sum_async(5, 7, 3) 